app/models/host/base.rb

```diff
@@ -321,6 +323,26 @@ def import_missing_ids
     # Provide _id aliases for consistency with the _name methods
     alias_attribute :hardware_model_id, :model_id
 
+    def locked?
```

Comment:
but for most use cases, I don't care when the host is getting unlocked. so it's a boolean method.